src/Microsoft.Health.Fhir.Core/Features/Operations/OperationParameterProperties.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations
+{
+    /// <summary>
+    /// Class for keeping track of the property names of the in/out parameter for each operation.
+    /// Some of these will be common across different operations and others might be specific.
```

Comment:
> Some of these will be common across different operations and others might be specific. [](start = 7, length = 87)

So far this is purely specific for DataConversion, right?
I would change to DataConverter/DataConversion Properties class, same way we have JobRecordProperties

Reply:
Agree.